scripts/release.py

```diff
@@ -240,11 +240,12 @@ def _calculate_next_due_date(self):
         return self.release_date + datetime.timedelta(days=14)
 
     def _generate_release_diff(self):
+        print("Generating release diff...")
```

Comment:
It might be nicer to use logging instead of print, it provides a bit more flexibility.